cub/log/log.cc

```diff
@@ -27,7 +27,8 @@ int threshold() {
 inline void LogStream::trace() const {
   char buf[32];
   timesystem().format(buf);
-  fprintf(stderr, "%s: %c %s:%d] %s\n", buf, "DIWEF"[level], file, line, str().c_str());
+  FILE* stream = freopen( "/home/john/Adlik/serving_time.log", "a", stdout);
```

Comment:
You could never known where the adlik_serving works. The log path  can't be assumed